### PR TITLE
add Questions page

### PR DESCRIPTION
**Pull request by hsnice16**
This PR will add the questions for all three categories.

Experience UX Live: https://marvelsquiz-questions.netlify.app/

This is how the Question page looks,
image

### CODE REVIEW

**Review by BalajiHarinath — APPROVED** [review]
Nice the UI is clean. Looks good to me.

**Review by Maianki — APPROVED** [review]
Looks good to merge.

File: pages/avengers-infinity-war/question-five.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    <link rel="stylesheet" href="../../app.css" />
+  </head>
+  <body>
+    <div class="container">
+      <nav class="fs-lg px-4 py-2">
+        <a class="text-primary-800" href="../../index.html"
+          ><span
+            class="brand-letter fs-xxl kushan-font-family mx-0p5 text-primary"
+            >m</span
+          >arvels<span
```
Comment (samyukthasudhakar) [inline pages/avengers-infinity-war/question-five.html:53]: The tags seem to be split up here. Formatting issue I suppose. There's other places this has occurred too, please take a look.

**Review by samyukthasudhakar — APPROVED** [review]
The changes look good to me !

File: pages/avengers-infinity-war/question-five.html (hunk 1 above)
Comment (hsnice16) [reply pages/avengers-infinity-war/question-five.html:53]: Okay, thanks for letting me know. I will check.

**Comment by hsnice16** [comment]
Thank you guys, for reviewing this.

**Pull request merged**